src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
+function convertToNumber(currency) {
+  return Number(currency.split(',').join('.').slice(1));
+}
```

Comment:
The function `convertToNumber` is not self-explanatory. Consider renaming it to something more descriptive, like `convertCurrencyToNumber`, so that it's clear what it does without having to read the implementation.